### PR TITLE
S03 us44 news page

### PR DESCRIPTION
**Pull request by AnassAmh**

### CODE REVIEW

File: src/app/components/header-mobile/header-mobile.component.html [hunk 1]
```diff
@@ -51,7 +51,7 @@
               </ul>
             </li>
   
-            <a routerLink="home"><li routerLink="home">News</li></a>
+            <a routerLink="/News"><li routerLink="News">News</li></a>
```
Comment (Kileana27) [inline src/app/components/header-mobile/header-mobile.component.html:54]: routerLink ="news"

File: src/app/components/header/header.component.html [hunk 2]
```diff
@@ -28,7 +28,7 @@
         </div>
         <div class="navbar-right">
           <ul>
-            <a routerLink="home"><li routerLink="home">News</li></a>
+            <a routerLink="/News"><li routerLink="News">News</li></a>
```
Comment (Kileana27) [inline src/app/components/header/header.component.html:31]: routerLink ="news"

File: src/app/news-page/news-page.component.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<body>
+    <div class="news-container">
+        <div class="carousel">
+            <app-carousel [slides]="slides"></app-carousel>
+        </div>
+    </div>
+
+    <div class="news-content">
+        <div class="news-item">
```
Comment (Kileana27) [inline src/app/news-page/news-page.component.html:9]: change div in art

File: src/app/news-page/news-page.component.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            <img src="assets/pictures/Image article.png" alt="News Image 1">
+            <div class="news-text">
+                <h2>News 1</h2>
+                <p>Lorem ipsum dolor sit amet. Sit autem nemo in minus quas nam itaque minima sed dicta atque.</p>
+            </div>
+        </div>
+        <div class="news-item">
```
Comment (Kileana27) [inline src/app/news-page/news-page.component.html:16]: change div in art

File: src/app/news-page/news-page.component.scss [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    display: flex;
+    flex-direction: column;
+}
+
+.news-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 20px;
+    //background-color: rgba(15, 15, 15, 0.7);
```
Comment (Kileana27) [inline src/app/news-page/news-page.component.scss:24]: dead code

File: src/app/carousel/carousel.component.html [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<div class="carousel">
```
Comment (Benja-san) [inline src/app/carousel/carousel.component.html:1]: ul tag

File: src/app/carousel/carousel.component.html [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+    <button (click)="previousSlide()">&#10094;</button>
+    <div class="carousel-slide" *ngFor="let slide of slides; let i = index" [class.active]="i === currentIndex"
```
Comment (Benja-san) [inline src/app/carousel/carousel.component.html:3]: li tag

File: src/app/components/header-mobile/header-mobile.component.html [hunk 8]
```diff
@@ -51,7 +51,7 @@
               </ul>
             </li>
   
-            <a routerLink="home"><li routerLink="home">News</li></a>
+            <a routerLink="news"><li routerLink="news">News</li></a>
```
Comment (Benja-san) [inline src/app/components/header-mobile/header-mobile.component.html:54]: a tag should be inside the li tag and not the opposite

File: src/app/news-page/news-page.component.html [hunk 9]
```diff
@@ -0,0 +1,24 @@
+<div id="back">
+    <div class="news-container">
+        <div class="carousel">
+            <app-carousel [slides]="slides"></app-carousel>
+        </div>
+    </div>
+
+    <article class="news-content">
```
Comment (Benja-san) [inline src/app/news-page/news-page.component.html:8]: section more than article

File: src/app/news-page/news-page.component.html
Comment (Benja-san) [inline src/app/news-page/news-page.component.html:1]: your file need a unique H1

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Review by Kileana27 — APPROVED** [review]

**Pull request merged**